### PR TITLE
chore(package): update to v1.0.0

### PR DESCRIPTION
**Pull request by asutedja**

### CODE REVIEW

**Comment by asutedja** [comment]
I messed up.  I published a `1.0.0` before checking to see if it can be merged or not.  Didn't rebase off of master.  I will now wait until this PR is merged before publishing.  However, as a result, I  unpublished  `1.0.0` and will now need to publish a `v1.0.1`.

**Review by 10xLaCroixDrinker — APPROVED** [review]

**Review by infoxicator — APPROVED** [review]

**Pull request merged**
